src/makeInfinityAdder.js

```diff
@@ -5,8 +5,24 @@
  * @return {function}
  */
 
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(initialValue = 0) {
+  let total = initialValue;
+
+  const fn = curr => {
+    if (curr === undefined) {
+      const result = total;
+
+      total = 0;
```

Comment:
Avoid modifying the total back to 0 after returning the result. It should retain its value for future additions.

Reply:
@AlexKherson ?